go/vt/vtgate/grpcvtgateservice/server.go

```diff
@@ -211,10 +211,10 @@ func (vtg *VTGate) ExecuteBatch(ctx context.Context, request *vtgatepb.ExecuteBa
 		sqlQueries[queryNum] = query.Sql
 		bindVars[queryNum] = bv
 	}
-	results, err = vtg.server.ExecuteBatch(ctx, sqlQueries, bindVars, request.Keyspace, request.TabletType, request.AsTransaction, request.Session, request.Options)
```

Comment:
I couldn't mark it as deprecated because PHP wouldn't build. But I've commented it to be so for now.